### ISSUE TITLE
New 9.11 install - packages outdated

### ISSUE THREAD

**Issue by eXistenZe** [dnnsoftware/Dnn.Platform#5346]
## Description of bug
Brand new install of DNN 9.11.
The extension manager shows this:
![image](https://user-images.githubusercontent.com/26518879/196695245-5e60636c-966f-4bb8-ae78-1278c68367d5.png)

![image](https://user-images.githubusercontent.com/26518879/196695290-89c3e6b3-7a0f-451a-8702-5cfc2b861581.png)

![image](https://user-images.githubusercontent.com/26518879/196695378-5a54ab5c-60f1-424e-b010-901e486c3be8.png)

## Steps to reproduce
Brand new install.

## Expected behavior
DNN installs latest versions of packages.

I have a couple other instalations, and this also happened to one other site (updated a couple times). The last is fine with all packages reporting latest version.

**Comment by jeremy-farrance** [comment]
I've noticed this subject keeps coming up. I also had the expectation that these Version/Upgrade numbers would match the latest install or upgrade - until (I believe) I was told otherwise and not to worry about it. If I am right and that it is normal and not a problem, _then the problem is_ that we need to fix the display of information so people's expectations are met.
 
What can we do on that screen so that an average DNN Joe like me clearly sees that those version numbers are okay? 
 
A big part of the problem appears to be the version numbers shown in the Upgrade column. Maybe that needs a rethink or rework? But seeing all those "9.3.2" versions is definitely counter-intuitive.
 
Again, I feel this needs some attention simply because I have repeatedly seen it waste time in both conversations, community discussions, and also people taking the time to report it as an issue.

**Comment by eXistenZe** [comment]
Many many improvements needed.
You can see bellow that this does not happen in all installs. Bellow is a DNN with the same version.
Then, there is the random numbers and bool in the updates on the right.
Finally, how can we tell if an upgrade package missed something or not?

![image](https://user-images.githubusercontent.com/26518879/196702351-1a50366b-b657-4e1d-9184-5ca753b4ba7c.png)

![image](https://user-images.githubusercontent.com/26518879/196702503-dafb80ac-1219-446f-b9f4-146c3fbf6629.png)

**Comment by sleupold** [comment]
in your web.config file, section "appSettings", can you please compate value for key "UpdateServiceUrl"

**Comment by eXistenZe** [comment]
<add key="UpdateServiceUrl" value="https://dnnplatform.io" />
On both

**Comment by sleupold** [comment]
are both apppool users able to access outside urls?

**Comment by eXistenZe** [comment]
Can you rephrase that question to a more n00b friendly version?

**Comment by eXistenZe** [comment]
Or is there an easy way for me to test that? One of the installs is a plesk
shared account.

**Comment by mitchelsellers** [comment]
We need to ensure that all core elements match expected version for clarity. 

The upgrade service is on my plate to try and find a way to "fix" rogue versions.

**Comment by donker** [comment]
I could not replicate the original reported issue. I have installed a clean 9.11.0 and all packages were labeled as 9.11.0. I cannot explain the behavior seen by the poster with certain packages being labeled as 9.3.2. There are no older versions within the distributed zip so I can't see where that might come from.

As to the funky versions on the right: this is due to an imperfection in our upgrade service and we hope to repair this one day. It has nothing to do with your own DNN installation. These are the versions echoed by our upgrade server and they can get thrown off sometimes. We regularly go in to reset those values to what they should be.

I'm closing this issue for now and I'd invite anyone to resubmit if this can be reproduced consistently.
